### PR TITLE
Support e2e on multiple java version

### PR DESCRIPTION
**Pull request by kevindiu**
SSIA

### CODE REVIEW

File: .github/workflows/pr-auto-merge.yml [hunk 1]
```diff
@@ -15,11 +15,11 @@ jobs:
...
   e2e:
-    uses: ./.github/workflows/_e2e.yml
-    needs:
-      - rebase
+    name: e2e
+    needs: rebase
+    uses: ./.github/workflows/_e2e_multi.yml
     with:
-      notify_slack_if_failed: true
+      notify_slack_if_failed: false
```
Comment (kevindiu) [inline .github/workflows/pr-auto-merge.yml:22]:
```suggestion
      notify_slack_if_failed: true
```

File: .github/workflows/pr-auto-merge.yml [hunk 2]
```diff
@@ -15,11 +15,11 @@ jobs:
         uses: cirrus-actions/rebase@1.8
 
   e2e:
-    uses: ./.github/workflows/_e2e.yml
-    needs:
-      - rebase
+    name: e2e
+    needs: rebase
```
Comment (kevindiu) [inline .github/workflows/pr-auto-merge.yml:19]:
```suggestion
    needs:
      - rebase
```

File: .github/workflows/pr-auto-merge.yml [hunk 3]
```diff
@@ -15,9 +15,10 @@ jobs:
         uses: cirrus-actions/rebase@1.8
 
   e2e:
-    uses: ./.github/workflows/_e2e.yml
+    name: e2e
```
Comment (hlts2) [inline .github/workflows/pr-auto-merge.yml:18]:
```suggestion
```

File: .github/workflows/pr.yml [hunk 4]
```diff
@@ -2,8 +2,10 @@ on:
   pull_request:
 
 jobs:
+  
   e2e:
-    uses: "./.github/workflows/_e2e.yml"
+    name: e2e
```
Comment (hlts2) [inline .github/workflows/pr.yml:7]:
```suggestion
```

**Review by kmrmt — APPROVED** [review]

**Review by hlts2 — APPROVED** [review]
LGTM 👍

**Pull request merged**